specification/azsadmin/resource-manager/backup/Microsoft.Backup.Admin/preview/2016-05-01/BackupLocations.json

```diff
@@ -0,0 +1,315 @@
+                    {
+                        "$ref": "Backup.json#/parameters/ResourceGroupParameter"
+                    },
+                    {
+                        "$ref": "#/parameters/BackupLocationParameter"
+                    },
+                    {
+                        "$ref": "Backup.json#/parameters/ApiVersionParameter"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "schema": {
+                            "$ref": "Backups.json#/definitions/Backup"
+                        }
+                    },
+                    "202": {
+                        "description": "ACCEPTED",
+                        "schema": {
+                            "$ref": "Backups.json#/definitions/Backup"
+                        }
+                    }
+                }
+            }
+        }
+    },
+    "definitions": {
+        "BackupLocation": {
+            "description": "Information about the backup location.",
+            "type": "object",
+            "properties": {
+                "properties": {
+                    "description": "Properties of a backup location.",
+                    "$ref": "#/definitions/BackupLocationModel",
+                    "x-ms-client-flatten": true
+                }
+            },
+            "allOf": [
+                {
+                    "$ref": "Backup.json#/definitions/Resource"
+                }
+            ]
+        },
+        "BackupLocationModel": {
+            "type": "object",
+            "description": "Properties of a backup location.",
+            "properties": {
+                "externalStoreDefault": {
+                    "description": "Information about an external storage location.",
+                    "$ref": "#/definitions/ExternalStore",
+                    "x-ms-client-flatten": true
+                }
+            }
+        },
+        "ExternalStore": {
+            "type": "object",
+            "description": "Information about an external storage location.",
+            "properties": {
+                "path": {
+                    "description": "Path to the update location",
+                    "type": "string"
+                },
+                "userName": {
+                    "description": "Username to access the location.",
+                    "type": "string"
+                },
+                "password": {
+                    "description": "Password to access the location.",
+                    "type": "string"
+                },
+                "encryptionKeyBase64": {
+                    "description": "Encryption key.",
+                    "type": "string"
+                },
+                "backupFrequencyInHours": {
+                    "description": "The interval, in hours, for the frequency that the scheduler takes a backup.",
+                    "type": "string"
```

Comment:
should this be  an integer like `backupRetentionPeriodInDays`?

Reply:
According to BRP API, this should be integer.

Reply:
Done.